clusterloader2/pkg/measurement/common/etcd_metrics.go

```diff
@@ -60,6 +60,13 @@ type etcdMetricsMeasurement struct {
 // - start - Starts collecting etcd metrics.
 // - gather - Gathers and prints etcd metrics summary.
 func (e *etcdMetricsMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	isSSHSupported := config.ClusterFramework.GetClusterConfig().IsSSHSupported
+	// Etcd is only exposed on localhost level. We are using ssh method
+	if isSSHSupported {
```

Comment:
`if !isSSHSupported` ?

Also, do we need to introduce this local variable? I don't think it improves readability a lot. What about
```
if !config.ClusterFramework.GetClusterConfig().IsSSHSupported {
  ...
}
```